app/models/assignment.rb

```diff
@@ -1235,7 +1235,7 @@ def current_submission_data(current_role)
         end
       end
 
-      base[:members] = member_info.pluck('users.user_name') unless member_info.nil?
+      base[:members] = member_info.pluck('users.user_name', 'roles.hidden') unless member_info.nil?
```

Comment:
Currently `members` is `nil` if there are no members. This leads to a runtime error when there's a group with no members. You can change this to set `members` to be an empty list when there's no members.